pkg/driver/target_path.go

```diff
@@ -0,0 +1,54 @@
+package driver
+
+import (
+	"errors"
+	"path/filepath"
+)
+
+var (
+	ErrInvalidTargetPath = errors.New("ParseTargetPath: Invalid target path")
+)
+
+// A TargetPath represents a parsed target path from Kubernetes.
+type TargetPath struct {
+	PodID    string
+	VolumeID string
+}
+
+// ParseTargetPath parses given target path from Kubernetes.
+// Target paths are generated by Kubernetes for CSI drivers to mount their filesystem.
+// They are in the form of "/var/lib/kubelet/pods/<pod-uuid>/volumes/kubernetes.io~csi/<volume-id>/mount".
+func ParseTargetPath(path string) (TargetPath, error) {
+	path, mount := filepath.Split(path)
+	if path == "" || mount != "mount" {
```

Comment:
Is it possible to put this all in a for loop popping off a list? It's very repetitive right now

Reply:
I feel like using a loop here would make it less readable. I can use a RegExp here, I think that would be more readable.

Reply:
Updated to use a RegExp with https://github.com/awslabs/mountpoint-s3-csi-driver/pull/251/commits/422d7322788e9ed3d0215bb6382c98c903eb6896. Seems more readable now